pkg/descheduler/evictions/evictions.go

```diff
@@ -128,7 +135,7 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptio
 	if pod.Spec.NodeName != "" {
 		if pe.maxPodsToEvictPerNode != nil && pe.nodepodCount[pod.Spec.NodeName]+1 > *pe.maxPodsToEvictPerNode {
 			if pe.metricsEnabled {
```

Comment:
Alternatively, profile name could be passed in `EvictOptions` since it's called by Profile.go [here](https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/framework/profile/profile.go#L62) and is already aware of profile name.

What do you think?

Reply:
Liked the idea, I was thinking about doing it. Got some extras related to the `strategyName` todo comments.